gen/templates/model.go

```diff
@@ -83,9 +83,12 @@ func (data {{camelCase .Name}}) getClassName() string {
 	return "{{.ClassName}}"
 }
 
-func (data {{camelCase .Name}}) toBody() nxos.Body {
+func (data {{camelCase .Name}}) toBody(update bool) nxos.Body {
```

Comment:
Can we maybe name this parameter "statusReplace" instead of "update" to better indicate the purpose? And maybe the same for the schema field: "strict_update" -> "status_replace". What do you think?